packages/openapi-generator/src/parser/document.ts

```diff
@@ -36,7 +36,7 @@ export function parseAllOperations(
         .filter(method => method in pathDefinition)
         .map(method =>
           // The line above makes sure that the pathDefinition[method] is not undefined
-          parseOperation(pattern, method, pathDefinition[method]!, refs)
+          parseOperation(pattern, pathDefinition, method, refs)
```

Comment:
The comment refers to this filter: `.filter(method => method in pathDefinition)` right? So perhaps put it directly to the line or above it.

Reply:
I removed the comment, it was outdated anyways.